scripts/train/train.py

```diff
@@ -473,6 +485,10 @@ def main(cfg: DictConfig) -> Trainer:
         tokenizer,
         device_train_batch_size,
     )
+
+    if mosaicml_logger:
```

Comment:
Let's move all the mosaicml logger logic here instead of instantiating in above and using it here. It'll be easier to maintain in one place.

Reply:
we're probably going to end up adding more events and moving the init back to the top shortly...so maybe just leave it there? either way is fine with me tho

Reply:
oh good point, i just thought it was this one event. If we add more than one event, I'd prefer having the instantiation separate actually